docs/content/config/security/mail_crypt.md

```diff
@@ -35,43 +44,24 @@ Official Dovecot documentation: https://doc.dovecot.org/configuration_manual/mai
           - ./certs/:/certs
         . . .
     ```
-4. While you're editing the docker-compose.yml, add the configuration files you copied out:
+
+5. While you're editing the docker-compose.yml, add the configuration file:
     ```yaml
     services:
       mailserver:
         image: docker.io/mailserver/docker-mailserver:latest
         volumes:
         . . .
-          - ./config/dovecot/20-lmtp.conf:/etc/dovecot/conf.d/20-lmtp.conf
-          - ./config/dovecot/20-imap.conf:/etc/dovecot/conf.d/20-imap.conf
-          - ./config/dovecot/20-pop3.conf:/etc/dovecot/conf.d/20-pop3.conf
+          - ./config/dovecot/10-custom.conf:/etc/dovecot/conf.d/10-custom.conf
           - ./certs/:/certs
         . . .
     ```
-5. The `mail_crypt` plugin, unless you're using a non-standard configuration of docker-mailserver, should be enabled on both `lmtp` and `imap`. You'll want to edit three different files:
-    - `./config/dovecot/20-lmtp.conf`
-      ```
-      protocol lmtp {
-        mail_plugins = $mail_plugins sieve mail_crypt
-        plugin {
-          mail_crypt_global_private_key = </certs/ecprivkey.pem
-          mail_crypt_global_public_key = </certs/ecpubkey.pem
-          mail_crypt_save_version = 2
-        }
-      }
-      ```
-    - `./config/dovecot/20-imap.conf`
-      ```
-      protocol imap {
-        mail_plugins = $mail_plugins imap_quota mail_crypt
-        plugin {
-          mail_crypt_global_private_key = </certs/ecprivkey.pem
-          mail_crypt_global_public_key = </certs/ecpubkey.pem
-          mail_crypt_save_version = 2
-        }
-      }
-      ```
-    - If you use pop3, make the same changes in `20-pop3.conf`
-6. Start the container and monitor the logs for any errors
-
-This should be the minimum required for encryption of the mail while in storage.
\ No newline at end of file
+
+6. Start the container, monitor the logs for any errors, send yourself a message, and then confirm the file on disk is encrypted:
```

Comment:
```suggestion
6. Start the container, monitor the logs for any errors, send yourself a message and then confirm the file on disk is encrypted:
```

Not sure if a comma is needed.

Reply:
As written, the comma is necessary before the and.